### ISSUE TITLE
train phase assert fail

### ISSUE THREAD

**Issue by IQ17** [vita-epfl/openpifpaf#170]
Hi, I followed the master branch readme.md, to train the model 
But I experienced the assert error, as below.

  File "/home/user/miniconda/envs/py36/lib/python3.6/site-packages/torch/nn/modules/module.py", line 541, in __call__
    result = self.forward(*input, **kwargs)
  File "../openpifpaf/openpifpaf/network/losses.py", line 228, in forward
    assert len(x) == 1 + 2 * self.n_vectors + self.n_scales

the training command is the following: 
ShuffleNet models are trained without ImageNet pretraining:

time CUDA_VISIBLE_DEVICES=0,1 python3 -m openpifpaf.train \
  --batch-size=64 \
  --basenet=shufflenetv2x2 \
  --head-quad=1 \
  --epochs=150 \
  --momentum=0.9 \
  --headnets pif paf paf25 \
  --lambdas 30 2 2 50 3 3 50 3 3 \
  --loader-workers=16 \
  --lr=0.1 \
  --lr-decay 120 140 \
  --no-pretrain \
  --weight-decay=1e-5 \
  --update-batchnorm-runningstatistics \
  --ema=0.03

**Comment by IQ17** [comment]
I think the error is due to the parameters of the paf headnet, but what are the correct parameters?

**Comment by LittleDonkey1203** [comment]
Have you solved the problem? I have the same issue.

**Comment by IQ17** [comment]
> Have you solved the problem? I have the same issue.

firstly, I tried to update pytorch to 1.3.1-gpu; then I used the release version here https://github.com/vita-epfl/openpifpaf/releases

**Comment by LittleDonkey1203** [comment]
I see, I also think that it may be caused by the edition difference, and now I change the internal code and successfully run it. But I’ll also try your suggestion, thank you!

**Comment by gd2016229035** [comment]
@LittleDonkey1203 I have the same issue. I try to updated both pytorch and pifpaf version as @IQ17 said, But it does not work. How you solve this problem? Thank you!

**Comment by zechendev** [comment]
> I see, I also think that it may be caused by the edition difference, and now I change the internal code and successfully run it. But I’ll also try your suggestion, thank you!

Hi, Can you share the solution? thank you!

**Comment by AIprogrammer** [comment]
Hi, will version 0.9 or 1.0 work? I met the same error. Do you know how to solve it now? Thanks. 

> > I see, I also think that it may be caused by the edition difference, and now I change the internal code and successfully run it. But I’ll also try your suggestion, thank you!
> 
> Hi, Can you share the solution? thank you!

Hi, will version 0.9 or 1.0 work? I met the same error. Do you know how to solve it now? Thanks.

**Comment by thanhtrung98** [comment]
I met the same error.
I solved 
You can use version 0.9.0

**Comment by AIprogrammer** [comment]
> I met the same error.
> I solved
> You can use version 0.9.0

Thanks for your reply. Version 0.9.0 output the 'unknown head to create an encoder: paf25' error. with pytorch 1.4.0. Can you share your running scripts?

**Comment by thanhtrung98** [comment]
> > I met the same error.
> > I solved
> > You can use version 0.9.0
> 
> Thanks for your reply. Version 0.9.0 output the 'unknown head to create an encoder: paf25' error. with pytorch 1.4.0. Can you share your running scripts?

paf25 didn't have in source code so error is true
<img width="262" alt="Screen Shot 2020-02-26 at 4 40 44 PM" src="https://user-images.githubusercontent.com/43095377/75332409-e9eae900-58b6-11ea-9c45-665958077fcd.png">

I use pytorch 1.4.0

time CUDA_VISIBLE_DEVICES=0,1,2,3 python -m openpifpaf.train \
  --lr=1e-4 \
  --momentum=0.95 \
  --epochs=75 \
  --lr-decay 120 140 \
  --batch-size=32 \
  --basenet=resnet50block5 \
  --head-quad=1 \
  --headnets pif paf \
  --square-edge=401 \
  --lambdas 30 2 2 50 3 3 \
  --regression-loss=laplace --loader-workers 16
I trained 4 GPU
